internal/pkg/gateway/commit/blockparser.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package commit
```

Comment:
Nit: The package name commit seems a bit over. Can we name it commitnotifications?

Reply:
That seems to contradict the general advice that Go package names are short (typically not multi-word) and avoid stuttering in struct names contained within that package (e.g. `commitnotifications.Notifier`): https://golang.org/doc/effective_go.html#package-names